app/serializers/dist/json_serializer.rb

```diff
@@ -2,9 +2,10 @@
 
 module Dist
   class JSONSerializer
-    def initialize(verticals:, properties:, version:)
+    def initialize(verticals:, properties:, mapping_rules: nil, version:)
```

Comment:
Do we need default value here?

```suggestion
    def initialize(verticals:, properties:, mapping_rules:, version:)
```